x-pack/plugins/siem/common/constants.ts

```diff
@@ -91,6 +91,8 @@ export const DETECTION_ENGINE_RULES_STATUS_URL = `${DETECTION_ENGINE_RULES_URL}/
 export const DETECTION_ENGINE_PREPACKAGED_RULES_STATUS_URL = `${DETECTION_ENGINE_RULES_URL}/prepackaged/_status`;
 
 export const TIMELINE_URL = '/api/timeline';
+export const TIMELINE_DRAFT_CLEAN_URL = `${TIMELINE_URL}/_draft/clean`;
```

Comment:
In follow up PR can we have the post act like a clean without the word clean in the URL.